packages/core/test/unit/html/NodeProcessor.test.js

```diff
@@ -123,6 +123,43 @@ test('processNode processes dropdown with header slot taking priority over heade
                            testData.PROCESS_DROPDOWN_HEADER_SLOT_TAKES_PRIORITY_EXPECTED);
 });
 
+// test('preprocessor converts inline colour syntax to the appropriate span', async () => {
```

Comment:
Could we repurpose the test cases here into unit tests for the plugin that you wrote? Quite a waste to remove and would be great to have these behaviors checked for the plugin

Reply:
Okay repurposed!